Extensions/one_click_postage.js

```diff
@@ -843,7 +843,11 @@ XKit.extensions.one_click_postage = new Object({
 	},
 
 	make_button_reblogged: function(m_button) {
-		m_button.addClass("reblogged");
+		if (XKit.page.react) {
+			m_button.find("svg").attr("fill", "var(--green)");
+		} else {
+			m_button.addClass("reblogged");
+		}
 	},
```

Comment:
this would be a nice opportunity to change `m_button` to `$button` to make it more clear it's a jquery argument